### ISSUE TITLE
Make ojdbc and postgresql optional on dbunit-core

### ISSUE THREAD

**Issue by hazendaz** [vasiliygagin/dbunit#55]
In the old dbunit, we were already excluding poi-ooxml, junit (4), postgresql (2 copies), and ojdbc8.

With new version, great, no legacy poi in the core, no junit in the core.  So less exclusions.

Now, can you make the drivers optional so user can pick.  That allows us to avoid exclusions.  Both oracle and postgresql?

**Comment by hazendaz** [comment]
as an FYI, I also have been supporting scriptable-dataset for past 7 years.  So I've applied your repo to replace the other.  While the other is technically still around its in sourceforge and anyone over there I feel treat their repos as end of the road.  At any rate, see https://github.com/hazendaz/scriptable-dataset/releases/tag/scriptable-dataset-2.4.0 to see I picked you up and I have made the two drivers provided there anyways.
